eclipse/sw360
# feat(svm): Publish SVM codes to Community

Signed-off-by: Smruti Prakash Sahoo <smruti.sahoo@siemens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

SVM code is an wrapper written over the sw360 community CVE service, SVM code enables sw360 server to connect with the SVM server, over the exposed SVM endpoints. Sw360 also uses SVM services to sync with the SVM server. Contributing the SVM specific code to community will benefit all the organization/s that uses SVM server.

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?

This feature is configurable via the flag `enable.security.vulnerability.monitoring` in sw360.properties. 
Put the SVM related configs in sw360.properties file like (`svm.components.url`,  `svm.components.vulnerabilities.url` etc.) and check if the SVM functionalities working fine or not.


> Have you implemented any additional tests?

### Checklist
Must:
